### PR TITLE
Next iteration Transitions

### PR DESCRIPTION
**Pull request by sdras**
Fixes some todos on the overview, and works out text for more sections.

### CODE REVIEW

File: src/guide/transitions-overview.md [hunk 1]
```diff
@@ -87,13 +87,29 @@ TODO: use example
 
 You may notice that the animations shown above are using things like `transforms`, and applying strange properties like `perspective`- why were they built that way instead of just using `margin` and `top` etc?
 
-There are ways that we can create buttery smooth animations on the web by being aware of performance. We want to hardware accelerate elements when we can, and use properties that don't trigger repaints. Let's go over some of how we can accomplish this.
+We can create extremely smooth animations on the web by being aware of performance. We want to hardware accelerate elements when we can, and use properties that don't trigger repaints. Let's go over some of how we can accomplish this.
+
+### Transform and Opacity
+
+We can check resources like [CSS-Triggers](https://csstriggers.com/) to see which properties will trigger repaints if we animate them. Here, if you look under `transform`, you will see:
```
Comment (phanan) [inline src/guide/transitions-overview.md:94]: TIL!

File: src/guide/transitions-overview.md [hunk 2]
```diff
@@ -87,13 +87,29 @@ TODO: use example
...
+Opacity behaves similarly. Thus, they are ideal candidates for movement on the web.
 
 ### Hardware Acceleration
 
-If you'd like to
+Properties such as `perspective`, `backface-visibility`, and `transform: translateZ(x)` will allow the browser to know you need hardware acceleration.
 
-TODO: finish writing
+If you wish to hardware accelerate an element you can apply any of these properties (not all are necessary, only one):
+
```
Comment (phanan) [inline src/guide/transitions-overview.md:105]: nitpick:

```suggestion
If you wish to hardware-accelerate an element, you can apply any of these properties (not all are necessary, only one):

```

File: src/guide/transitions-overview.md [hunk 3]
```diff
@@ -87,13 +87,29 @@ TODO: use example
...
+```css
+perspective: 1000px;
+backface-visibility: hidden;
+transform: translateZ(0);
+```
+
+Many JS libraries like GreenSock will assume you want hardware acceleration and will apply them by default, you do not need to set them manually.
```
Comment (phanan) [inline src/guide/transitions-overview.md:112]: question: Do you think we should link to GreenSock's website here? (Sorry if it has been linked in one of the previous paragraphs).
nitpick: 

```suggestion
Many JS libraries like GreenSock will assume you want hardware acceleration and will apply them by default, so you do not need to set them manually.
```

File: src/guide/transitions-overview.md [hunk 4]
```diff
@@ -128,17 +144,70 @@ If we were to apply these states to a transition, it would look something like t
...
+  }
+}
 
-Take this bounce, for instance. In CSS we have to declare each keyframe, up and down. In JavaScript, we can express all of that movement within the ease, by declaring `bounce` in the GreenSock API (other JS libraries have other types of easing defaults).
+.bounceInDown {
+  animation-name: bounceInDown;
+}
+```
+
+And here is the same bounce in JS using [GreenSock (GSAP)](https://greensock.com/):
```
Comment (phanan) [inline src/guide/transitions-overview.md:204]: nitpick: I think the link to GreenSock should be added the first time we mention it instead.

File: src/guide/transitions-overview.md [hunk 5]
```diff
@@ -128,17 +144,70 @@ If we were to apply these states to a transition, it would look something like t
...
 
-TODO: add example
+```js
+gsap.from(element, { duration: 1, ease: 'bounce.out', y: -500 })
+```
 
-We'll be using [GreenSock (GSAP)](https://greensock.com/) in some of the examples in the sections following, they have a great [ease visualizer](https://greensock.com/ease-visualizer) that will help you build nicely crafted eases.
+We'll be using GreenSock in some of the examples in the sections following, they have a great [ease visualizer](https://greensock.com/ease-visualizer) that will help you build nicely crafted eases.
```
Comment (phanan) [inline src/guide/transitions-overview.md:210]: nitpick: The text sounds like they can be two sentences to me (sorry I missed the original PR).

```suggestion
We'll be using GreenSock in some of the examples in the sections following. They have a great [ease visualizer](https://greensock.com/ease-visualizer) that will help you build nicely crafted eases.
```

File: src/guide/transitions-overview.md (hunk 4 above)
Comment (sdras) [reply src/guide/transitions-overview.md:204]: Oh, I thought I did, probably added more material, I'll fix it right up. Thanks!

Comment (sdras) [reply src/guide/transitions-overview.md]: Ok all done :)

**Pull request merged**